inst/report/summary_report.Rmd

```diff
@@ -6,53 +6,40 @@ params:
   output_zip: NA
   spectrum_file: NA
   options: NA
+  lang: NA
 output:
   html_document:
+    css: styles.css
     includes:
       in_header: header.html
     anchor_sections: false
 ---
 
-``` {css, echo = FALSE}
+::: {#translate lang="en"}
+text in english
+:::
 
-.main-container {
-max-width: 1300px;
-margin-left: 0;
-margin-right: 1;
-text-align: justify;
-text-justify: inter-word;
-}
-
-.banner {
- background-color: #b5132c;
-  font-weight: 200;
-  position: relative;
-  padding: .5rem 1rem;
-  display: flex;
-max-width: 1300px;
-
-}
-
-.logo_naomi {
- height: 60px;
- padding-left: 15px;
- padding-right: 10px;
-}
-
-.logo_unaids {
-   height: 60px;
-}
-
-.text {
- padding-left: 10px;
- padding-top: 10px;
- color: white;
-font-size: 35px;
+::: {#translate lang="fr"}
+texte en français
+:::
 
+```{r, include = FALSE}
+if (params$lang == "fr") {
+  print_en <- FALSE
+  print_fr <- TRUE
+} else {
+  print_en <- TRUE
+  print_fr <- FALSE
 }
+```
 
+```{r include = print_en, eval = print_en}
```

Comment:
Use `include = FALSE` to hide the code block and `eval = FALSE` to stop the code block being evaluated. i.e. to stop evaluation and hide printing both should be `FALSE`